### ISSUE TITLE
Fix missing alt attributes on images in CTAGrids

### ISSUE THREAD

**Issue by soshwosh** [lsst-epo/rubin-obs-client#1133]
Some of the most visited pages on the main site are missing alt text for images in CTAGrids, lowering our ADA compliance scores. Do some light discovery on the CTAGrid component to determine whether the issue lies in our code or if it's an external issue (for example, from wherever we're fetching the asset, it simply was not given an alt value) and add the alt values if possible.

[A&D](https://docs.google.com/document/d/1yhpdc8VyHK8z4kOPnOtM1Lip1iJZhrHI2w7cFcrFEmE/edit?tab=t.0#heading=h.gpy9s3bh2exm)

**Comment by soshwosh** [comment]
This is actually an issue with the assets in Craft not being given alt text. The same accessibility failure on the images used in the heroes on the `/about` and `/about/vera-rubin` pages are caused by the same issue. Possible technical solution: update Craft configs to make alt text a required field. Needs discussion with content creators.

**Comment by soshwosh** [comment]
Content creators are on the fence about making the field required as this is a task that gets gets deprioritized when bandwidth is low. For now, I've compiled a list of the most impactful images to add alt text to, and handed it off.
